src/Model/ROI.py

```diff
@@ -424,9 +424,16 @@ def get_pixluts(dict_ds):
     non_img_type = ["rtdose", "rtplan", "rtss"]
     for ds in dict_ds:
         if ds not in non_img_type:
-            img_ds = dict_ds[ds]
-            pixlut = calculate_matrix(img_ds)
-            dict_pixluts[img_ds.SOPInstanceUID] = pixlut
+            if isinstance(ds, str):
+                if ds[0:3] != 'sr-':
+                    img_ds = dict_ds[ds]
+                    pixlut = calculate_matrix(img_ds)
+                    dict_pixluts[img_ds.SOPInstanceUID] = pixlut
+            else:
+                img_ds = dict_ds[ds]
+                pixlut = calculate_matrix(img_ds)
+                dict_pixluts[img_ds.SOPInstanceUID] = pixlut
+
```

Comment:
I think you can merge the if statements. That will reduce the duplicated codes. (429-431 and 433-435).

Reply:
The code was written this way under the assumption that if the `if` statements are merged, e.g. `if (isinstance(ds, str)) and (ds[0:3] == 'sr-')`, and ds wasn't a string, then there would be a type error with the second operand. I did this as I wasn't sure if Python checked both operands, or just the first. However upon getting this comment I had a look at the Python documentation, which states:

> The expression `x and y` first evaluates _x_; if _x_ is false, its value is returned; otherwise, _y_ is evaluated and the resulting value is returned.

https://docs.python.org/3/reference/expressions.html?highlight=boolean#boolean-operations

So I think the if statements can safely be merged. Thank you for bringing this up! Thanks for bringing up the "Export Clinical Data" option in the toolbar too. Just a simple oversight that will also be fixed.